core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -235,6 +249,12 @@ class TradeEvaluation {
             }
         }
     }
+    fun evaluateDistanceSellCost(civInfo: CivilizationInfo, capitalcity: CityInfo, city: CityInfo): Int{
```

Comment:
This too should be renamed to something more appropriate. This doesn't calculate sell cost, but rather LESS sell cost. Something like evaluateSellDiscountForCitiesFarFromCapital.

Reply:
Seems a little big but sure